python/pysmurf/client/debug/smurf_noise.py

```diff
@@ -639,7 +639,7 @@ def noise_vs(self, band, var, var_range, meas_time=30,
                         np.max(self.get_amplitude_scale_array(band)),dtype=int))
                 self.run_serial_gradient_descent(band)
                 self.run_serial_eta_scan(band)
-                self.tracking_setup(band,lms_freq_hz=self.lms_freq_hz[band],
+                self.tracking_setup(band,lms_freq_hz=self._lms_freq_hz[band],
```

Comment:
This isn't introduced here, but since we're reviewing now - Don't know we want `tracking_setup` to use the lms_freq that the user found before? It should probably load the existing `lms_freq`.